specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/alerts.json

```diff
@@ -561,6 +561,11 @@
           "readOnly": true,
           "type": "string",
           "description": "Azure resource ID of the workspace that the alert was reported to."
+        },

```

Comment:
minor: could you indent this property the same number of spaces as the other properties are?

Reply:
caused due to the interpretation of tabs to spaces in the ide, fixed now.